src/button/Button.module.css

```diff
@@ -61,6 +61,17 @@ limitations under the License.
   outline: auto;
 }
 
+.button:disabled,
+.toolbarButton:disabled,
+.toolbarButtonSecondary:disabled,
+.iconButton:disabled,
+.iconCopyButton:disabled,
+.secondary:disabled,
+.secondaryHangup:disabled,
+.copyButton:disabled {
```

Comment:
Do we need to list these all? Shouldn't they all have class of `.button`?

Reply:
good question. this is copied from the focus field above?

Reply:
also works with just button.
The question is, if we really want all buttons to behave like this...

Reply:
I think for now to just have this style for the toolbarButton might be the best option to avoid style regressions.